src/components/ec-modal/ec-modal.vue

```diff
@@ -156,10 +156,20 @@ function hasPositiveButton() {
 function hasNegativeButton() {
   return !!slots.negative;
 }
+
+// zIndex
```

Comment:
We know from the variable name that this is a zIndex, I think the comment is not necessary here. We could add some comments to separate and unit some parts of code from time to time, in those cases when we want to declare some blocks of code and it is not obviously without comments.

Reply:
yes the comment is also needed just to separate the code's groups